wsd-frontend/src/components/wsd/Notifications/Notifications.tsx

```diff
@@ -60,20 +62,64 @@ export function Notifications({ hasNew }: { hasNew?: boolean }) {
     }
   }
 
+  async function markAllAsRead() {
+    return await wsd.markAllNotificationsAsRead()
+  }
+
+  function onMarkAllAsRead() {
+    markAllAsRead()
+      .then((data) => {
+        const { error } = data
+        if (error) {
+          const errors = error as { detail: string }
+          toast(errors.detail, {
+            description: 'There was an error marking notifications as read.',
+          })
+          return
+        }
+
+        setNotifications((prev) => {
+          return prev.map((notification) => ({
+            ...notification,
+            is_read: true,
+          }))
+        })
+        toast(data.data?.message || 'Notifications marked as read')
+        setHasNewState(false)
+      })
+      .catch((error) => {
+        console.error('Error marking notifications as read:', error)
+        toast('Error marking notifications as read', {
+          description: 'There was an error marking notifications as read.',
+        })
+      })
+  }
+
   return (
     <Overlay breakpoint="md" onOpenChange={onOverlayOpenChange}>
       <OverlayTrigger asChild>
         <Button variant="ghost" className="flex gap-2 h-10 w-10 rounded-full p-2">
           <div className="relative">
             <Icons.Bell size={20} />
-            {hasNew && <span className="absolute -top-0.5 -right-0.5 h-3 w-3 rounded-full bg-red-600 border" />}
+            {hasNewState && <span className="absolute -top-0.5 -right-0.5 h-3 w-3 rounded-full bg-red-600 border" />}
           </div>
         </Button>
       </OverlayTrigger>
       <OverlayContent align="end" side="bottom">
         <OverlayTitle className="hidden">Notifications</OverlayTitle>
         <OverlayDescription className="hidden">Notifications</OverlayDescription>
         <ScrollArea className="max-h-[50vh] overflow-auto">
+          {!loading && hasNewState && (
+            <Button
+              className="flex items-center gap-2 w-full"
+              variant={'outline'}
```

Comment:
This is on the nitpicky side of things but unless necessary can you avoid curly brackets around tsx tag attributes? 
`variant='outline'` instead of `variant={'outline'}`

Reply:
Yeah i forgot to remove them will do. Isnt there a eslint option for this?